client/src/main/java/io/confluent/kafka/schemaregistry/client/rest/entities/requests/TagSchemaRequest.java

```diff
@@ -106,6 +109,26 @@ public void setRuleSet(RuleSet ruleSet) {
     this.ruleSet = ruleSet;
   }
 
+  @JsonProperty("rulesToMerge")
+  public RuleSet getRulesToMerge() {
+    return rulesToMerge;
+  }
+
+  @JsonProperty("rulesToMerge")
+  public void setRulesToMerge(RuleSet rulesToMerge) {
```

Comment:
Why not take List<Rule> as a parameter?  Or should we call it setRuleSetToMerge?

Reply:
+1

Reply:
`rulesToMerge` takes a `RuleSet`, and `rulesToRemove` takes a set of rule names.  By "rules" I'm referring to the object semantically, not to the data structure that is passed in.  I'd rather not call it `ruleSetToMerge` and `rulesToRemove`

Reply:
IMO I think  `rulesToRemove` is better than `ruleNamesToRemove`. But I think `ruleSetToMerge` might better than `rulesToMerge`. However, it seems like the field that the customer sets is `rulesToMerge`. What do you think about changing that json field to `ruleSetToMerge` since it has the same attributes as the existing `ruleSet` field?

Reply:
I'd rather keep it `rulesToMerge`.  In any case, this API may be temporary.